src/tools/sizedecrease/sizedecreasetool.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright(c) 2017-2019 Alejandro Sirgo Rica & Contributors
```

Comment:
Do we really need to add the license header to every file? =) Also the date seems wrong.

GPL only recommends that.

https://www.gnu.org/licenses/gpl-faq.html#NoticeInSourceFile

> You should put a notice at the start of each source file, stating what license it carries, in order to avoid risk of the code's getting disconnected from its license. If your repository's README says that source file is under the GNU GPL, what happens if someone copies that file to another program? That other context may not show what the file's license is. It may appear to have some other license, or no license at all (which would make the code nonfree). Adding a copyright notice and a license notice at the start of each source file is easy and makes such confusion unlikely.This has nothing to do with the specifics of the GNU GPL. It is true for any free license.

But it doesn't sound like a good reason, because if someone is okay with violating the license they can just remove the header when copying. :thinking: 

And if they are not intending to violate, then they should just check where this file came from, not sure how it could get disconnected so severely that it would be difficult. It’s always possible just to google the class name, etc. if everything else is lost somehow. :)

Reply:
Hmm you are right - should be discussed if we do that in the future. Will ask everybody in Slack.